transformato/bin/simulation-rsfe_charmm.sh

```diff
@@ -2,26 +2,16 @@
 
 
 path=$1
-SWITCH=$2
+
+source  /opt/intel/oneapi/setvars.sh
 
 cd ${path}
 pwd
 hostname
 
 
 input=charmm_run_vacuum
-charmm_openmm_domdec -i ${input}.inp > log_vac.out
+OMP_NUM_THREADS=8 charmm_c47_omm_domdecgpu -i ${input}.inp > log_vac.out
 
 input=charmm_run_waterbox
```

Comment:
I guess it's fine to be that specific. I don't see many other people using charmm-domdec anyways.